2DW28WSQJ/note.json

```diff
@@ -0,0 +1,290 @@
+        "forms": {}
+      },
+      "results": {
+        "code": "SUCCESS",
+        "msg": [
+          {
+            "type": "TABLE",
+            "data": "1\n85\n"
+          }
+        ]
+      },
+      "apps": [],
+      "jobName": "paragraph_1540521857472_161462042",
+      "id": "20181026-024417_1685384875",
+      "dateCreated": "2018-10-26 02:44:17.472",
+      "dateStarted": "2018-10-31 01:34:07.650",
+      "dateFinished": "2018-10-31 01:34:08.285",
+      "status": "FINISHED",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "title": "Test Zeppelin Spark ",
+      "text": "%spark \n\nimport spark.implicits._\nval primitiveDS \u003d Seq(1, 2, 3).toDS()\nprimitiveDS.map(_ + 1).collect() \n\n",
+      "user": "splice",
+      "dateUpdated": "2018-10-31 01:34:08.350",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "scala",
+          "editOnDblClick": false,
+          "completionKey": "TAB",
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/scala",
+        "title": true
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
+        "code": "SUCCESS",
+        "msg": [
+          {
+            "type": "TEXT",
+            "data": "import spark.implicits._\nprimitiveDS: org.apache.spark.sql.Dataset[Int] \u003d [value: int]\nres10: Array[Int] \u003d Array(2, 3, 4)\n"
+          }
+        ]
+      },
+      "runtimeInfos": {},
+      "apps": [],
+      "jobName": "paragraph_1540927379744_-306541172",
+      "id": "20181030-192259_1985721246",
+      "dateCreated": "2018-10-30 19:22:59.744",
+      "dateStarted": "2018-10-31 01:34:08.380",
+      "dateFinished": "2018-10-31 01:34:09.555",
+      "status": "FINISHED",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "title": "Test pySpark",
+      "text": "%spark.pyspark\nfrom splicemachine.spark.context import PySpliceContext\n\ndef get_jdbc_url():\n    import os\n    framework \u003d os.environ[\u0027FRAMEWORK_NAME\u0027]\n    return \u0027jdbc:splice://{framework}-proxy.marathon.mesos:1527/splicedb\u0027.format(framework\u003dframework)\n\nsplice \u003d PySpliceContext(get_jdbc_url(), spark)\n\n\n# Query Features\nquery_results \u003d splice.df(\"select * from SYS.SYSTABLES\")\nquery_results.show(10)\n",
+      "user": "splice",
+      "dateUpdated": "2018-10-31 10:41:15.261",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "python",
+          "editOnDblClick": false,
+          "completionKey": "TAB",
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/python",
+        "title": true
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
```

Comment:
results should be cleared in all 'check' paragraphs of this notebook by default, else it gives the impression that something was checked and passed

Reply:
Done